src/data.js

```diff
@@ -141,39 +141,45 @@ Namespace.ensureNamespace = function(namespace, expires) {
 };
 
 /**Delete expired entries */
-Namespace.expireEntries = function(parent, indexedTask, continuationToken=null) {
-  console.log(`expireEntries in '${parent}' with token '${continuationToken}'`);
-  return this.query({
-    parent: parent,
+Namespace.expireEntries = function(indexedTask, continuationToken=null) {
+  //console.log(`expireEntries in '${parent}' with token '${continuationToken}'`);
+  return this.scan({
   },
   {
     limit:         500,
     continuation:   continuationToken,
-  }).then(async (data) => { 
+  }).then(async (data) => {
+    console.log('..namespaces', data);
     var dataLength = data.entries.length;
+    now = new Date();
+    now = new Date(
+      now.getFullYear(),
+      now.getMonth(),
+      now.getDate(),
+      0, 0, 0, 0
+    );
+    now = now.getTime();
```

Comment:
The queue uses `taskcluster.fromNow` to calculate this.

https://github.com/taskcluster/taskcluster-queue/blob/a2007b195c0cd875b64ea29372731ccd5e760fc5/src/main.js#L614

Note that it also passes the value into the expires function.  That makes testing a little easier.